packages/core/src/vite-env.d.ts

```diff
@@ -7,11 +7,13 @@ declare module 'vite/types/customEvent' {
     'motion-canvas:meta': {source: string; data: any};
     'motion-canvas:meta-ack': {source: string};
     'motion-canvas:export': {
-      frame: number;
       data: string;
-      mimeType: string;
-      project: string;
-      isStill: boolean;
+      meta: {
+        name: string;
+        subDirectories: string[];
+        mimeType: string;
+        frameNumber: number;
+      };
```

Comment:
I'm not sure what's the point of wrapping these things in an additional object.

Reply:
changed it to a flat structure now.